lib/core/ui/dashboard/home/home_favorites_card.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter/material.dart';
+
+class DashboardFavoritesCard extends StatelessWidget {
```

Comment:
What's the purpose of this widget? Please remove it if you only instantiate an empty Card widget

Reply:
It's a placeholder card for the favorites tab, which I'm about to do soon.